dist/api_data.py

```diff
@@ -512,6 +512,11 @@ def __ge__(self, other):
             versions of WiredTiger. These operations are informational and
             skipped in recovery.''',
             type='boolean'),
+        Config('slow_checkpoint', 'false', r'''
+            if true, each time checkpoint visits an internal page it will
+            sleep for 10000 ms.
```

Comment:
I'd be inclined to not mention a specific time in here, rather make it a bit more generic. Something like: "if true, slow down checkpoint creation by slowing down internal page processing"